bench.py

```diff
@@ -2,12 +2,45 @@
 import os
 import sys
 import time
+import psutil
+import json
+import urllib
+import urllib2
+from datetime import datetime
 
 import click
 import statistics
+import scrapy
 
 
-def calculator(test, arg, n_runs, only_result, workpath=os.getcwd()):
+CODESPEED_URL = 'http://localhost:8000/'
+
+current_date = datetime.today()
+
+
+def get_latest_commit(owner, repo):
+    url = 'https://api.github.com/repos/{owner}/{repo}/commits?per_page=1'.format(
+        owner=owner, repo=repo)
+    response = urllib2.urlopen(url).read()
+    data = json.loads(response.decode())
+    return data[0]
+
+
+def get_env():
+    pyversion = sys.version.rsplit(" ")[0]
+    scrapyversion = float(''.join(str(e)
+                                  for e in list(scrapy.version_info))) / 100
```

Comment:
I think this might break with 1.4.10 or 1.10.1, although this is more theoretical currently. What is the purpose of this function, what should it return, and why ``'.'.join(map(str, scrapy.version_info))`` does not work?